Python/Anime-tracker/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Anime tracker  in python
+python script which will give info about searched anime such as :
+- Tags (Genre)
+- Rating
+- Total episodes released till date
+- Active years
+# Packages Used
+- bs4 (for web scraping)
+-  requests 

```

Comment:
If possible add hyperlinks to the PyPi Packages. Remove the Web Scraping reference.